src/feasibility_form_nls.jl

```diff
@@ -85,7 +85,34 @@ function FeasibilityFormNLS(nls :: FeasibilityResidual)
                      )
   nls_meta = NLSMeta(nequ, nvar, x0=[meta.x0; zeros(nequ)], nnzj=nequ, nnzh=0)
 
-  nlp = FeasibilityFormNLS(meta, nls_meta, nls, NLSCounters())
+  nlp = FeasibilityFormNLS{FeasibilityResidual}(meta, nls_meta, nls, NLSCounters())
```

Comment:
In this constructor, there is no `name` kwarg. So if I specify a name, I'll fall back on the default constructor.